src/nvim/CMakeLists.txt

```diff
@@ -311,17 +312,21 @@ add_custom_command(OUTPUT ${GENERATED_UNICODE_TABLES}
 add_custom_command(
   OUTPUT ${GENERATED_API_DISPATCH} ${GENERATED_FUNCS_METADATA}
          ${API_METADATA} ${MSGPACK_LUA_C_BINDINGS}
+         ${GENERATED_API_DOC_METADATA}
   COMMAND ${LUA_PRG} ${API_DISPATCH_GENERATOR} ${CMAKE_CURRENT_LIST_DIR}
                      ${GENERATED_API_DISPATCH}
                      ${GENERATED_FUNCS_METADATA} ${API_METADATA}
                      ${MSGPACK_LUA_C_BINDINGS}
+                     ${GENERATED_API_DOC_METADATA}
+                     ${PROJECT_SOURCE_DIR}/runtime/doc/api.mpack
                      ${API_HEADERS}
   DEPENDS
     ${API_HEADERS}
     ${MSGPACK_RPC_HEADERS}
     ${API_DISPATCH_GENERATOR}
     ${GENERATOR_C_GRAMMAR}
     ${CMAKE_CURRENT_LIST_DIR}/api/dispatch_deprecated.lua
+    ${PROJECT_SOURCE_DIR}/runtime/doc/api.mpack
```

Comment:
checking in the mpack file is a bit unfortunate but I suppose it avoids adding python as a build dependency?

Reply:
`gen_api_dispatch` needs access to that file, so yeah, would probably have to have python if I didn’t check in the mpack file, so that the file could be generated first.